vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1257,6 +1258,89 @@ public Call modifyVideoInAlbums(@NotNull final Video video,
 
     // </editor-fold>
 
+    // -----------------------------------------------------------------------------------------------------
+    // Connected Apps
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="Connected Apps">
+
+    /**
+     * Gets a {@link com.vimeo.networking2.ConnectedAppList} for the authorized user.
+     *
+     * @param cacheControl The cache control.
+     * @return A {@link Call} object that can be used to request a {@link com.vimeo.networking2.ConnectedAppList}.
+     */
+    @NotNull
+    public Call<com.vimeo.networking2.ConnectedAppList> getConnectedApps(@NotNull final CacheControl cacheControl) {
+        return mVimeoService.getConnectedApps(getAuthHeader(), createCacheControlString(cacheControl));
+    }
+
+
+    /**
+     * Returns a {@link Call} object that can be used to request a {@link com.vimeo.networking2.ConnectedApp}.
+     *
+     * @param type         The {@link ConnectedAppType} of the {@link com.vimeo.networking2.ConnectedApp} to return.
+     * @param cacheControl The cache control.
+     * @return A {@link Call} object that can be used to request a {@link com.vimeo.networking2.ConnectedApp}.
+     */
+    @NotNull
+    public Call<com.vimeo.networking2.ConnectedApp> getConnectedApp(@NotNull final ConnectedAppType type,
```

Comment:
Minor: Add an import for the `ConnectedApp` so the class doesn't have to use the fully qualified name here or in the docs above.